setup.py

```diff
@@ -1,11 +1,15 @@
 #!/usr/bin/env python
 
 from os.path import exists
+import re
 from setuptools import setup
-import unification
+
+contents = re.findall(
+    r'__version__ = \'([.0-9]+)\'',
+    open('unification/__init__.py', 'r').read())[0]
```

Comment:
> In more recent projects I've just been duplicating the version string in __init__.py and setup.py. It's not the most elegant solution but it does work pretty simply.

My experience trying that approach has been that it's really easy to forget the `__init__` version string, though it can work with careful code review. I'm fine with that approach if you don't like the `os.path.join` solution.